modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -4521,12 +4521,38 @@ definition {
 	}
 
 	@description = "LPS-139603 - Verify that the Object Name is displayed on the Relationship tab when a Relationship is created"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ObjectNameDisplayedOnTheRelationshipTab {
+		for (var letter : list "A,B") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object ${letter}",
+				objectName = "CustomObject${letter}",
+				pluralLabelName = "Custom Objects ${letter}");
+		}
+
+		for (var letter : list "A,B") {
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldLabelName = "Custom Field ${letter}",
+				fieldName = "customObjectField${letter}",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject${letter}");
+		}
+
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObjectA",
+			objectName_2 = "CustomObjectB",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.openObjectAdmin();
 
-		// TODO LPS-141681 ObjectNameDisplayedOnTheRelationshipTab pending implementation
+		ObjectPortlet.selectCustomObject(label = "Custom Object A");
+
+		ObjectAdmin.gotoRelationshipsTab();
 
+		CreateObject.assertRelatedObject(relatedObject = "CustomObject");
```

Comment:
Hi @andreefarias 

You need specific the object correct, I believe that you need to use `CustomObjectB`

Reply:
Fixed!